_posts/2022-06-06-adaptive-batch-scheduler.md

```diff
@@ -116,7 +121,7 @@ To solve this problem, we need to make the subpartitions evenly consumed by down
 Note that this is a temporary solution, the ultimate solution would be the [Auto-rebalancing of workloads](#auto-rebalancing-of-workloads), which may come soon.
 
 ## Build up execution graph dynamically
-Before Flink 1.15, the execution graph was fully built in a static way before starting scheduling. To allow parallelisms of job vertices to be decided lazily, the execution graph must be able to be built up dynamically.
+Before the introduction of adaptive batch scheduler, the execution graph was fully built in a static way before starting scheduling. To allow parallelisms of job vertices to be decided lazily, the execution graph must be able to be built up dynamically.
```

Comment:
the introduction of adaptive batch scheduler -> adaptive batch scheduler was introduced to Flink

`introduction` might be misleading

Reply:
Fixed